editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -6165,7 +6168,7 @@ CanvasItemEditorViewport::CanvasItemEditorViewport(EditorNode *p_node, CanvasIte
 	label->add_color_override("font_color_shadow", Color(0, 0, 0, 1));
 	label->add_constant_override("shadow_as_outline", 1 * EDSCALE);
 	label->hide();
-	editor->get_gui_base()->add_child(label);
+	p_controls->add_child(label);
```

Comment:
I think the previous solution was better. It's better to avoid adding parameters for which it is hard to understand what they do. If you need to access a specific control managed by the editor, you can add another function to the editor API.
That should also allow plugins to add their own controls there.

Reply:
Makes perfect sense! I'll add a function instead of passing the extra control.